build/SandBoxAndPackage.proj

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(RepoRoot)/Directory.Build.props" />
+
+  <PropertyGroup>
+    <SandBoxFolderName>bin/WorkingCopy</SandBoxFolderName>
+    <RelativeSandBoxPackageOutputFolder>$(SandBoxFolderName)/RelativeSandBoxPackageOutputFolder</RelativeSandBoxPackageOutputFolder>
+    <SandBoxRepoRoot>$(RepoRoot)/$(SandBoxFolderName)/$(LinuxDistrosNeedNativeInstaller)</SandBoxRepoRoot>
+    <NugetCacheFolderName>.nuget</NugetCacheFolderName>
+    <RelativeCompileOutputPath>bin/2</RelativeCompileOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_MirrorOfRepoRootForTools Include="$(RepoRoot)/**/*.*" Exclude="$(RepoRoot)/$(SandBoxFolderName)/**/*.*;$(RepoRoot)/nuget/**/*.*;$(RepoRoot)/bin/**/*.*" />
+    <_ArtifactsToPackage Include="$(RepoRoot)/$(RelativeCompileOutputPath)/$(Rid)/**/*.*"/>
+    <_ArtifactsToPackage Include="$(RepoRoot)/bin/Directory.Build.props"/>
+    <_ToolsCacheToSpeedUp Include="$(RepoRoot)/$(RelativeCLIBuildBinaries)/**/*.*"/>
+    <_NugetCacheToSpeedUp Include="$(RepoRoot)/$(NugetCacheFolderName)/**/*.*" />
+  </ItemGroup>
+
+  <Target Name="SandBoxAndPackage" DependsOnTargets="PackageWithDocker;CopyPackageResult"/>
+
+  <Target Name="PrepareSandBox" >
+    <Copy
+      SourceFiles="@(_MirrorOfRepoRootForTools)"
+      DestinationFiles="@(_MirrorOfRepoRootForTools -> '$(SandBoxRepoRoot)/%(RecursiveDir)%(Filename)%(Extension)')"/>
+
+    <Exec Command="sh -c 'cd $(SandBoxRepoRoot) &amp;&amp; git clean -fxd'"/>
+
+    <Copy
+      SourceFiles="@(_ArtifactsToPackage)"
+      DestinationFiles="@(_ArtifactsToPackage -> '$(SandBoxRepoRoot)/$(RelativeCompileOutputPath)/$(Rid)/%(RecursiveDir)%(Filename)%(Extension)')"/>
+
+    <Copy
+      SourceFiles="@(_ToolsCacheToSpeedUp)"
+      DestinationFiles="@(_ToolsCacheToSpeedUp -> '$(SandBoxRepoRoot)/$(RelativeCLIBuildBinaries)/%(RecursiveDir)%(Filename)%(Extension)')"/>
+
+    <Copy
+      SourceFiles="@(_NugetCacheToSpeedUp)"
+      DestinationFiles="@(_NugetCacheToSpeedUp -> '$(SandBoxRepoRoot)/$(NugetCacheFolderName)/%(RecursiveDir)%(Filename)%(Extension)')"/>
+
+  </Target>
+
+  <Target Name="PackageWithDocker" DependsOnTargets="PrepareSandBox">
+    <PropertyGroup>
+      <CommandToInvokeBuildScriptInDockerToPackageInSandBox>$(SandBoxRepoRoot)/build.sh --configuration $(BuildConfiguration) --docker $(DockerFolder) --skip-prereqs /target:GenerateInstallersAndCopyOutOfSandBox /p:RelativeSandBoxPackageOutputFolder=$(RelativeSandBoxPackageOutputFolder)</CommandToInvokeBuildScriptInDockerToPackageInSandBox>
```

Comment:
Nit: this is a super long line.  You could break it up into several lines by adding arguments, something like this:

```xml
<Command>build.sh --configuration $(BuildConfiguration)/<Command>
<Command>$(Command) -additionalParameter1 value1</Command>
<Command>$(Command) -additionalParameter2 value2</Command>
```

Reply:
Good trick!

Reply:
Done, fixed